web/src/client/mixins.js

```diff
@@ -259,12 +255,12 @@ const WithValidation = (superclass, object_path) => class extends superclass {
     let errors;
```

Comment:
What about renaming `errors` to something like `response`? Basically to avoid the `errors.errors.map` below.